### PR TITLE
OG-3980 Add Bulgarian and Norwegian to crowdin mapping

### PR DESCRIPTION
**Pull request by nabaroa**
📎 https://transferwise.atlassian.net/browse/OG-3980

Adds Bulgarian and Norwegian to crowdin mapping

## Checklist
- [ ] I have considered the impact of this change and added a [Change Classification](
https://transferwise.atlassian.net/wiki/spaces/EKB/pages/1401189673/Change+Classifications+and+Expectations) Label (`change:standard`, `change:impactful`, `change:emergency`)
- [ ] Change meets or does not compromise the [Baseline Security Requirements](https://transferwise.atlassian.net/wiki/spaces/EKB/pages/434929973/Baseline+Security+Requirements)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@transferwise/public-navigation",
-  "version": "8.1.6",
+  "version": "8.1.7",
```
Comment (karanjthakkar) [inline package.json:3]: Shall we do 8.2.0 instead since this is a "feature"?

**Review by karanjthakkar — APPROVED** [review]
tiny nit but looks good otherwise. What's the state of translations for these two languages?

**Comment by danlaush** [comment]
@karanjthakkar Both look like they're only missing text for the business nav, which is ok as we don't have those pages in these languages yet. Will only be using the Personal Nav for now.

File: crowdin.yml [hunk 2]
```diff
@@ -25,6 +26,7 @@ files: [
       "it": "it",
       "ja": "ja",
       "ko": "ko",
+      "nb": "nb",
```
Comment (danlaush) [inline crowdin.yml:29]: Annoying crowdin thing - tl;dr this line should be `"no": "nb",`

crowdin has language options for 
- Norwegian
- Norwegian Nynorsk
- Norwegian Bokmål

Unfortunately "plain Norwegian" doesn't exist. You're either writing in Nynorsk or Bokmal, and our content is in Bokmal.

**Review by danlaush — CHANGES_REQUESTED** [review]

File: crowdin.yml (hunk 2 above)
Comment (karanjthakkar) [reply crowdin.yml:29]: Wow. TIL.

Comment (nabaroa) [reply crowdin.yml]: I'd heard you mention this but didn't know. How amazing are the language puzzles.

"Today there are two official forms of written Norwegian, Bokmål (literally "book tongue") and Nynorsk ("new Norwegian"), each with its own variants. Bokmål developed from the Dano-Norwegian koiné language that evolved under the union of Denmark–Norway in the 16th and 17th centuries, while Nynorsk was developed based upon a collective of spoken Norwegian dialects. Norwegian is one of the two official languages in Norway. The other is Sámi, spoken by some members of the Sámi people, mostly in the Northern part of Norway. Norwegian and Sámi are not mutually intelligible, as Sámi belongs to the Finno-Ugric group of languages. Sámi is spoken by less than one percent of people in Norway. "

**Review by danlaush — APPROVED** [review]

**Pull request merged**